offchain/rollups-events/src/rollups_outputs.rs

```diff
@@ -9,7 +9,7 @@ use crate::{rollups_stream::decl_broker_stream, Address, Hash, Payload};
 
 decl_broker_stream!(RollupsOutputsStream, RollupsOutput, "rollups-outputs");
 
-/// Cartesi  output
```

Comment:
If there is a lot of formatting, I agree.
However, this is a typo, a single deleted character.
I don't think we should create a single commit for that. Otherwise, people will just see these things and not change them. Creating multiple commits and managing them is time consuming and can be a hassle. Imagine I find another single character typo, but I already have other commits enqueued. I would have to rebase/edit the first commit, do some git manipulation and then get back to what I was doing. This greatly hinders efficiency.
But them again, I agree we should separate **big** formatting refactors in different commits.